src/core.ts

```diff
@@ -58,9 +63,11 @@ export function activate() {
       state = State.Activated;
     } else {
       teardown();
+      return;
     }
   } catch (e) {
     onActivateError(e);
+    return;
```

Comment:
>return [](start = 4, length = 6)

What does adding these fix? Seems unrelated to format change.

Reply:
It is unrelated to the format change, but it was a small fix that I just corrected on the fly, while working on this PR :P

Essentially each of these catch blocks means that we've ran into a critical problem. These added 'return' statements make sure we terminate immediately after handling the problem

---
In reply to: [144390879](https://github.com/Microsoft/clarity-js/pull/39#discussion_r144390879) [](ancestors = 144390879)

Reply:
I think this branch will *not* be merged for a while. Can you make these same fixes to the main (beta) version?

---
In reply to: [144425231](https://github.com/Microsoft/clarity-js/pull/39#discussion_r144425231) [](ancestors = 144425231,144390879)

Reply:
Published a separate PR for that

---
In reply to: [144539040](https://github.com/Microsoft/clarity-js/pull/39#discussion_r144539040) [](ancestors = 144539040,144425231,144390879)